contracts/contracts/echidna/EchidnaTestMintBurn.sol

```diff
@@ -0,0 +1,130 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "./EchidnaDebug.sol";
+import "./EchidnaTestAccounting.sol";
+
+contract EchidnaTestMintBurn is EchidnaTestAccounting {
+    // Minting 0 tokens should not affect account balance
+    function testMintZeroBalance(uint8 targetAcc) public {
+        address target = getAccount(targetAcc);
+
+        uint256 balanceBefore = ousd.balanceOf(target);
+        mint(targetAcc, 0);
+        uint256 balanceAfter = ousd.balanceOf(target);
+
+        assert(balanceAfter == balanceBefore);
+    }
+
+    // Burning 0 tokens should not affect account balance
+    function testBurnZeroBalance(uint8 targetAcc) public {
+        address target = getAccount(targetAcc);
+
+        uint256 balanceBefore = ousd.balanceOf(target);
+        burn(targetAcc, 0);
+        uint256 balanceAfter = ousd.balanceOf(target);
+
+        assert(balanceAfter == balanceBefore);
+    }
+
+    // Minting tokens must increase the account balance by at least amount
+    //
+    // testMintBalance(uint8,uint256): failed!💥
+    //   Call sequence:
+    //       changeSupply(1)
+    //       testMintBalance(0,1)
+    //
+    //   Event sequence:
+    //       Debug(«balanceBefore», 0)
+    //       Debug(«balanceAfter», 0)
+    //
+    function testMintBalance(uint8 targetAcc, uint256 amount)
+        public
+        hasKnownIssue
+        hasKnownIssueWithinLimits
+    {
+        address target = getAccount(targetAcc);
+
+        uint256 balanceBefore = ousd.balanceOf(target);
+        uint256 amountMinted = mint(targetAcc, amount);
+        uint256 balanceAfter = ousd.balanceOf(target);
+
+        Debugger.log("amountMinted", amountMinted);
+        Debugger.log("balanceBefore", balanceBefore);
+        Debugger.log("balanceAfter", balanceAfter);
+
+        assert(balanceAfter >= balanceBefore + amountMinted);
```

Comment:
should we test that it is within the expected rounding limits? So that it is not too high from expected amount?